arch/x86_64/mcount-dynamic.c

```diff
@@ -216,6 +219,17 @@ void mcount_arch_find_module(struct mcount_dynamic_info *mdi,
 		if (sym->name[0] == '_')
 			continue;
 
+		/*
+		 * there might be some chances of not having patchable section
+		 * '__patchable_function_entries' but shows the NOPs pattern.
+		 * this can be treated in the same way.
+		 */
+		if (!memcmp(code_addr, patchable_gcc_nop, CALL_INSN_SIZE) ||
+		    !memcmp(code_addr, patchable_clang_nop, CALL_INSN_SIZE)) {
+			mdi->type = DYNAMIC_PATCHABLE;
```

Comment:
Maybe we can set it as `DYNAMIC_FENTRY_NOP` and check the NOP pattern differently?

Reply:
Is there any reason to do this? I actually don't get your point.

Reply:
If the section is not there, no need to special case the binary.  I think we can handle it same as fentry + nop.  It should only be handled separately if it comes with the section, right?